.build.yml

```diff
@@ -1,13 +1,32 @@
-image: archlinux
+image: alpine/edge
 oauth: pages.sr.ht/PAGES:RW
 sources:
   - https://github.com/swaywm/swaywm.org
 packages:
   - hut
+  - mandoc
+  - sway-doc
```

Comment:
I wonder if we should clone the sway repo instead and run scdoc, to grab the latest version even if the package is lagging or something?

Downsides: would need to checkout the latest tag to avoid publishing docs for unreleased versions, and would need to hardcode man pages paths.

Reply:
I think that's more effort that it's worth. The sway-docs package on alpine/edge should not take more than a day or two to update, and this saves us having to build all of sway.

Maybe if sway had some mechanism to instruct meson to _only_ build docs?

Reply:
> I think that's more effort that it's worth. The sway-docs package on alpine/edge should not take more than a day or two to update, and this saves us having to build all of sway.

Not a fan on relying on a distro to just build a man page, but I can live with that…

> Maybe if sway had some mechanism to instruct meson to only build docs?

One way would be to invoke scdoc directly. Another would be to pass the list of man pages to ninja (`ninja -C build/ sway.1 …`), but we'd still need to pull all sway build deps.

Reply:
TBH, I can also just fetch the sway repo, find pages with `find . -name '*.scd` and then run scdoc on them individually. Should be quite doable in a single sh script.